src/constructs/bindings.md

```diff
@@ -27,7 +30,9 @@ Any valid Grain functions.
 
 ## Description
 
-`let` allows you to introduce names, and `let rec` allows you to introduce recursive definitions.
+* `let` allows you to introduce names.
+* `let rec` allows you to introduce recursive definitions.
+* `let mut` allows you to introduce names with mutable values.
```

Comment:
Just because this hasn't been looked at in like two years 😛 :
```suggestion
* `let` allows you to introduce new names with values that never change. In other words, `let` allows you to create **immutable** variables.
* `let mut` allows you to introduce names with values that can change. In other words, `let mut` allows you to create **mutable** variables.
* `let rec` allows you to declare recursive functions.
```